web/tests/visualization.spec.ts

```diff
@@ -454,10 +477,31 @@ test('visualization context menu', async () => {
   const visualization = workspace.getStateView('vz0', 'visualization').visualization;
   const editor = workspace.getVisualizationEditor('vz0');
   const name = editor.left.vertexAttribute('name');
-  const gender = editor.left.vertexAttribute('gender');
-  const income = editor.left.vertexAttribute('income');
-  const age = editor.left.vertexAttribute('age');
-  const location = editor.left.vertexAttribute('location');
-  const weight = editor.left.edgeAttribute('weight');
-  const comment = editor.left.edgeAttribute('comment');
+
+  await name.visualizeAs('label');
+  await visualization.svg.locator('text=Eve >> .. >> .touch').click();
```

Comment:
What does it do?

Reply:
> What does it do?

Find `.touch` within the parent of something that has text `Eve`. Because the text itself is not clickable. The nodes have a dedicated `circle.touch` for interaction.

The `..` syntax is [Playwright shorthand](https://playwright.dev/docs/other-locators#parent-element-locator) for `xpath=..`.